tech/languages/ruby/gems-installation.md

```diff
@@ -47,7 +47,9 @@ To install `thor` gem you therefore install `rubygem-thor` package:
 
 ```
 $ sudo dnf install rubygem-thor
+```
```

Comment:
I think we need a blank like  before/after code blocks, otherwise all will display inline. I will fix it.